LoopKitUI/Views/ScheduleEditor.swift

```diff
@@ -351,7 +349,22 @@ struct ScheduleEditor<Value: Equatable, ValueContent: View, ValuePicker: View, A
         .disabled(tableDeletionState != .disabled || scheduleItems.count >= scheduleItemLimit)
     }
 
-    private func beginSaving() {
+    private func startSaving() {
+        guard mode == .settings || mode == .legacySettings else {
+            self.continueSaving()
+            return
+        }
+        
+        authenticate(authenticationChallengeDescription) {
+            switch $0 {
+            case .success: self.continueSaving()
+            case .failure(let error): self.settingSaveAlert = .saveError(error)
```

Comment:
I'll continue to push back on presenting an error here. All error states are communicated by the auth modal. If the user pushes cancel, they should not see a modal that tells them they just cancelled. I weighed in on the Slack thread too; seems that Pete agrees we don't need to handle the presentation of an error here.

Reply:
Okay, I will remove it.  That said, would you be okay with my leaving this new `SettingSaveAlert` in place as a generally-usable facility?

Reply:
Yeah, I don't see a problem with that

Reply:
;) I got rid of it.  Sorry...Please take another look, @mpangburn ?